CONTRIBUTING.md

```diff
@@ -140,7 +140,9 @@ $ npm install
 $ npm start
 ```
 
-This will set up everything needed to run the page on [`localhost:3000`](http://localhost:3000/). You can now modify the content of the [`/docs`](https://github.com/webdriverio/webdriverio/tree/main/docs) files as well as change styles and templates. The page will be automatically updated. If you add documentation in other places, you have to rerun the `npm start` script to re-generate the docs.
+This will set up everything needed to run the page on [`localhost:3000`](http://localhost:3000/). If you need to run on a different host or port, pass them as additional arguments to npm start, like `-- --host 0.0.0.0`.
```

Comment:
I added a line about changing host or port. I can remove if you don't like it. I run as much of my development environment in containers and so the default localhost didn't work and I needed to dig into docusaurus to learn how to configure it and figured this might save other people that same time